packages/frontend/core/src/blocksuite/block-suite-editor/starter-bar.tsx

```diff
@@ -16,6 +16,7 @@ import {
   EdgelessIcon,
   TemplateColoredIcon,
 } from '@blocksuite/icons/rc';
+import { effect } from '@preact/signals-core';
```

Comment:
is signal used in affine?

Reply:
should I expose a signal `doc.isEmpty$` and modify the effect ?
```ts
  useEffect(() => {
      return doc.isEmpty$.subscribe(empty => {
         setIsEmpty(empty);
      })
  }, [doc]);
```

Reply:
better